app/controllers/index.js

```diff
@@ -48,17 +48,17 @@ if (accounts.hasAccount()) {
 }
 
 (function () {
-    var Rating = require('Piwik/App/Rating');
-    var rating = new Rating();
-    rating.countLaunch();
+    //var Rating = require('Piwik/App/Rating');
+    //var rating = new Rating();
+    // rating.countLaunch();
```

Comment:
removed this and asking for tracking consent as it wasn't working fully anymore. the dialog was shown before any window was opened and then basically hidden. We won't really need it anymore